src/main/scala/cz/alenkacz/db/postgresscala/ConfigParser.scala

```diff
@@ -19,10 +19,11 @@ object ConfigParser {
   private final val PoolMaxQueueSize = "maxQueueSize"
   private final val PoolValidationInterval = "validationInterval"
 
-  private val defaultConfig = URLParser.DEFAULT
+  private val defaultConfiguration = URLParser.DEFAULT
+  private val defaultConfig = ConfigFactory.defaultReference().getConfig("postgres-connection-defaults")
```

Comment:
I don't see difference between defaultConfig and defaultConfiguration - maybe we can find better name for one of those?